docs/en/operations/settings/settings.md

```diff
@@ -5430,3 +5430,9 @@ Enabling this setting can lead to incorrect result as in case of evolved schema
 :::
 
 Default value: 'false'.
+
+## first_day_of_week
```

Comment:
a) please add a section "Possible values"
b) please move this to a more reasonable place than the end of the file, e.g. to after `date_time_overflow_behavior`